schema/daqconf/triggergen.jsonnet

```diff
@@ -47,6 +48,15 @@ local cs = {
 
   tc_readout_map: s.sequence( "tc_readout_map", self.tc_readout),
 
+  hsi_input: s.record("hsi_input", [
+    s.field("signal",         types.count, default=1, doc="HSI candidate maker accepted HSI signal ID"),
+    s.field("tc_type_name",   self.tc_type_name, default="kTiming", doc="Name of the TC type"),
+    s.field("time_before",    self.readout_time, default=1000, doc="Time to readout before TC time [ticks]"),
+    s.field("time_after",     self.readout_time, default=1001, doc="Time to readout after TC time [ticks]"),
```

Comment:
I know this isn't something you have changed, by why is the time_after 1001 (instead of 1000) (I remember there was a reason (probably a integtest expecting some exact size?))